src/subdomains/supporting/support-issue/entities/support-issue.entity.ts

```diff
@@ -28,12 +29,18 @@ export class SupportIssue extends IEntity {
   @Column({ length: 256, nullable: false })
   reason: SupportIssueReason;
 
+  @Column({ length: 256, nullable: true })
+  name: string;
+
   @Column({ length: 'MAX', nullable: true })
   description: string;
```

Comment:
Remove description as well (later)?

Reply:
Yeah maybe, last time we talked about it, we said it could be maybe needed for something, but yeah if not we can remove it 👍